### PR TITLE
mochi-margo: add v0.16.0, v0.17.0

### PR DESCRIPTION
**Pull request by mdorier**
Adds versions 0.16.0 and 0.17.0 to mochi-margo.

### CODE REVIEW

**Review by alecbcs — APPROVED** [review]
Verified Checksums. Looks good to me. Waiting on final approval from @carns.

**Review by carns — APPROVED** [review]

**Pull request merged**
